hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestScannerRetriableFailure.java

```diff
@@ -68,7 +68,7 @@
   private static final String FAMILY_NAME_STR = "f";
   private static final byte[] FAMILY_NAME = Bytes.toBytes(FAMILY_NAME_STR);
 
-  @Rule public TestTableName TEST_TABLE = new TestTableName();
+  @Rule public TableNameTestRule TEST_TABLE = new TableNameTestRule();
```

Comment:
Also not yours, but this variable is not `static` and its name really as no business being SCREAM_CASE. Bleh.

Reply:
Done.